src/VisualStudio/Core/Impl/CodeModel/ExternalElements/ExternalCodeProperty.cs

```diff
@@ -123,23 +116,61 @@ public EnvDTE80.vsCMOverrideKind OverrideKind
         {
             get
             {
-                throw new NotImplementedException();
+                var symbol = PropertySymbol;
+                var result = EnvDTE80.vsCMOverrideKind.vsCMOverrideKindNone;
+
+                if (symbol.IsAbstract)
+                {
+                    result = EnvDTE80.vsCMOverrideKind.vsCMOverrideKindAbstract;
+                }
+
+                if (symbol.IsVirtual)
+                {
+                    result = EnvDTE80.vsCMOverrideKind.vsCMOverrideKindVirtual;
+                }
+
+                if (symbol.IsOverride)
+                {
+                    result = EnvDTE80.vsCMOverrideKind.vsCMOverrideKindOverride;
+                }
+
+                if (symbol.IsSealed)
+                {
+                    result = EnvDTE80.vsCMOverrideKind.vsCMOverrideKindSealed;
+                }
+
+                return result;
             }
 
             set
             {
-                throw new NotImplementedException();
+                throw Exceptions.ThrowEFail();
```

Comment:
Fixed typo in my message above, you *can't* set properties for metadata elements. This now throws E_FAIL like other property setters in this file.